community/consistency-check/src/main/java/org/neo4j/consistency/ConsistencyCheckTool.java

```diff
@@ -72,23 +76,24 @@ public static void main( String[] args ) throws IOException
     }
 
     private final ConsistencyCheckService consistencyCheckService;
-    private final StoreRecoverer recoveryChecker;
     private final GraphDatabaseFactory dbFactory;
     private final PrintStream systemError;
     private final ExitHandle exitHandle;
+    private StoreRecoverer recoveryChecker;
+    private FileSystemAbstraction fs;
 
     ConsistencyCheckTool( ConsistencyCheckService consistencyCheckService, PrintStream systemError )
     {
-        this( consistencyCheckService, new StoreRecoverer(), new GraphDatabaseFactory(), systemError,
+        this( consistencyCheckService, new GraphDatabaseFactory(), new DefaultFileSystemAbstraction(), systemError,
                 ExitHandle.SYSTEM_EXIT );
     }
```

Comment:
You can have a private method in the test to clean it up. I prefer not having constructors only for tests.